abi-bindings/go/Teleporter/upgrades/TeleporterRegistry/packing.go

```diff
@@ -0,0 +1,81 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package teleporterregistry
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/subnet-evm/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+)
+
+var teleporterRegistryEntryType abi.Type
+
+type TeleporterRegistryEntry struct {
+	Entry              ProtocolRegistryEntry `json:"protocolRegistryEntry"`
+	DestinationAddress common.Address        `json:"destinationAddress"`
+}
+
+func init() {
+	// Create an ABI binding for TeleporterRegistryEntry, defined in TeleporterRegistry.sol
```

Comment:
Is `TeleporterRegistryEntry` an actual type defined in the contract? I think what we're packing here is the packed `ProtocolRegistryEntry` and `DestinationAddress` [here](https://github.com/ava-labs/teleporter/blob/main/contracts/src/Teleporter/upgrades/TeleporterRegistry.sol#L101).

I think we should define the ABI spec for `protocolRegistryEntryType` to be consistent with the actual contract type, then add helpers to extend to `teleporterRegistryEntryType`

Reply:
updated to match `protocolRegistryEntryType` and chagned function as a helper `PackTeleporterRegistryWarpPayload`